cpp/iedriver/BrowserFactory.cpp

```diff
@@ -971,12 +1053,14 @@ BOOL CALLBACK BrowserFactory::FindChildWindowForProcess(HWND hwnd, LPARAM arg) {
   } else {
     DWORD process_id = NULL;
     ::GetWindowThreadProcessId(hwnd, &process_id);
-    if (process_window_info->dwProcessId == process_id) {
+    LOG(DEBUG) << "Looking for " << process_window_info->dwProcessId;
+    // stahon, this needs to not happen or look for the right PID for anaheim
+    //if (process_window_info->dwProcessId == process_id) {
```

Comment:
Delete the commented out lines entirely.

Reply:
Fixed.